scripts/debug/pty-capture.py

```
@@ -0,0 +1,78 @@
+    p.add_argument("--delay", type=float, default=0.6,
+                   help="各送信の前に待つ秒数")
+    p.add_argument("cmd", nargs=argparse.REMAINDER)
+    args = p.parse_args()
+
+    cmd = args.cmd[1:] if args.cmd and args.cmd[0] == "--" else args.cmd
+    if not cmd:
+        p.error("コマンドが指定されていません")
+
+    pid, fd = pty.fork()
+    if pid == 0:
+        os.execvp(cmd[0], cmd)
```

Ruff (0.16.0)

[error] Starting a process without a shell

(S606)